db/seed/MessageContract.csv

```diff
@@ -1,4 +1,4 @@
-Id;KafkaTopicId;MessageType;Description;Example;Schema;Status;CreatedAt;CreatedBy;ModifiedAt;ModifiedBy
-AC4E4454-A30A-4731-989F-CA478D8AD979;269C3064-E1AD-49A8-8FE8-781E04A51303;something-created;Something has been created - cool!;{"foo": "bar"};{"fooschema": "barschema"};Provisioned;2023-02-10T08:00:00;SeedScript;2023-02-10T08:00:00;SeedScript
-16A8159E-F61F-431D-9A22-D3BF03F4FBD2;269C3064-E1AD-49A8-8FE8-781E04A51303;something-updated;Someone updated something - yay!;{"foo2": "bar2"};{"foo2schema": "bar2schema"};In Progress;2023-02-10T08:00:00;SeedScript;2023-02-10T08:00:00;SeedScript
-c867bc57-3c2b-4d8f-927b-0ffe445471f4;269C3064-E1AD-49A8-8FE8-781E04A51303;something-failed;Someone has tried to provision this contract but it failed;{foo3: bar3};{foo3schema: bar3schema};Failed;2023-02-10 08:00:00.000000;SeedScript;2023-12-28 09:45:24.263458;krijens@dfds.com
+Id;KafkaTopicId;MessageType;Description;Example;Schema;Status;CreatedAt;CreatedBy;ModifiedAt;ModifiedBy;SchemaVersion
+AC4E4454-A30A-4731-989F-CA478D8AD979;269C3064-E1AD-49A8-8FE8-781E04A51303;something-created;Something has been created - cool!;{"foo": "bar"};{"fooschema": "barschema"};Provisioned;2023-02-10T08:00:00;SeedScript;2023-02-10T08:00:00;SeedScript;1
+16A8159E-F61F-431D-9A22-D3BF03F4FBD2;269C3064-E1AD-49A8-8FE8-781E04A51303;something-updated;Someone updated something - yay!;{"foo2": "bar2"};{"foo2schema": "bar2schema"};In Progress;2023-02-10T08:00:00;SeedScript;2023-02-10T08:00:00;SeedScript;1
```

Comment:
Should these not have incremented version numbers?
They are  all referencing the same KafkaTopicId and the description indicate the desire to update  the existing rather than create additional new.